src/Akeneo/Category/back/Infrastructure/Symfony/Resources/config/handlers.yml

```
@@ -5,6 +5,6 @@ services:
       - '@Akeneo\Category\Application\Query\FindCategoryByCode'
       - '@Akeneo\Category\Application\Applier\UserIntentApplierRegistry'
       - '@event_dispatcher'
-      - '@Akeneo\Category\Application\Storage\ProcessCategoryUpdateMock'
+      - '@Akeneo\Category\Application\Storage\Save\ProcessCategorySave'
```

```suggestion
      - '@Akeneo\Category\Application\Storage\Save\CategorySaverProcessor'
```

to be persitent with the CategorySaverRegistry in the same namespace.

---

done